man/busctl.xml

```diff
@@ -144,6 +144,17 @@
         <xi:include href="version-info.xml" xpointer="v242"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>wait</command> <arg choice="opt"><replaceable>SERVICE</replaceable></arg> <arg choice="plain"><replaceable>OBJECT</replaceable></arg> <arg choice="plain"><replaceable>INTERFACE</replaceable></arg> <arg choice="plain"><replaceable>SIGNAL</replaceable></arg></term>
+
+        <listitem><para> Wait for a signal. Takes an object path, interface
```

Comment:
Not a matter, but unnecessary space at the beginning.
Also, please rebreak lines in this paragraph. Nowadays we accept about ~109 ch per line.